typescript/sdk/src/deploy/core/AbacusCoreChecker.ts

```diff
@@ -108,8 +108,12 @@ export class AbacusCoreChecker<
     validatorManager: MultisigValidatorManager,
   ): Promise<void> {
     const config = this.configMap[remote];
+
     const validatorManagerConfig = config.validatorManager;
-    const expectedValidators = validatorManagerConfig.validators;
+    const expectedValidators =
+      local !== remote && config.remove
+        ? []
+        : validatorManagerConfig.validators;
```

Comment:
In retrospect I guess we could also have just removed the validators from the config...

Reply:
lol yeah

Reply:
expected should always equal config